### PR TITLE
refpolicy: add cdrom-daemon policy

### PR DESCRIPTION
**Pull request by jean-edouard**
OXT-847

Signed-off-by: Jed <lejosnej@ainfosec.com>

### CODE REVIEW

File: recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff [hunk 1]
```diff
@@ -445,15 +447,15 @@ Index: refpolicy/policy/modules/contrib/xen.te
  
  manage_files_pattern(xenstored_t, xenstored_tmp_t, xenstored_tmp_t)
  manage_dirs_pattern(xenstored_t, xenstored_tmp_t, xenstored_tmp_t)
-@@ -430,6 +532,7 @@ files_pid_filetrans(xenstored_t, xenstor
+@@ -430,6 +534,7 @@ files_pid_filetrans(xenstored_t, xenstor
  manage_dirs_pattern(xenstored_t, xenstored_var_log_t, xenstored_var_log_t)
  append_files_pattern(xenstored_t, xenstored_var_log_t, xenstored_var_log_t)
  create_files_pattern(xenstored_t, xenstored_var_log_t, xenstored_var_log_t)
 +manage_files_pattern(xenstored_t, xenstored_var_log_t, xenstored_var_log_t)
```
Comment (dpsmith) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff]: What does this have to do with cdrom-daemon policy?

Comment (jean-edouard) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff]: Nothing, this is context.

Comment (dpsmith) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff]: If this isn't specific to the cdrom-daemon policy, should this not be in a separate commit to provide better traceability to policy changes?

Comment (dpsmith) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff]: Never mind, it was getting colored green which made me think it was a change not context.
